### PR TITLE
Security definition documentation updates

### PR DESCRIPTION
**Pull request by darrelmiller**

### CODE REVIEW

**Comment by fehguy** [comment]
@darrelmiller looks good to me!

File: versions/3.0.md [hunk 1]
```diff
@@ -3004,12 +3004,12 @@ Field Name | Type | Validity | Description
 <a name="securitySchemeDescription"></a>description | `string` | Any | A short description for security scheme.
 <a name="securitySchemeName"></a>name | `string` | `apiKey` | **Required.** The name of the header or query parameter to be used.
 <a name="securitySchemeIn"></a>in | `string` | `apiKey` | **Required** The location of the API key. Valid values are `"query"` or `"header"`.
-<a name="securitySchemeScheme"></a>scheme | `string` | `http` | **Required.** The name of the HTTP Authorization scheme to be used in the Authorization header as per RFC 7234.
-<a name="securitySchemeBearerFormat"></a>bearerFormat | `string` | `http` (`"bearer"`) | A hint to the client to identify how the bearer token should be formatted.
-<a name="securitySchemeOpenIdConnectUrl"></a>openIdConnectUrl | `string` | `openIdConnect` | **Required.** OpenId Connect URL to discover OAuth2 configuration values. 
+<a name="securitySchemeScheme"></a>scheme | `string` | `http` | **Required.** The name of the HTTP Authorization scheme to be used in the Authorization header as per [RFC 7235](https://tools.ietf.org/html/rfc7235).
```
Comment (DavidBiesack) [inline versions/3.0.md]: or link to the section:
the `[Authorization header of RFC 7235](https://tools.ietf.org/html/rfc7235#section-4.2)`

File: versions/3.0.md [hunk 2]
```diff
@@ -3004,12 +3004,12 @@ Field Name | Type | Validity | Description
...
+<a name="securitySchemeOpenIdConnectUrl"></a>openIdConnectUrl | `string` | `openIdConnect` | **Required.** OpenId Connect URL to discover OAuth2 configuration values. Url may be relative to location of OpenAPI document.
 <a name="securitySchemeFlow"></a>flow | `string` | `oauth2` | **Required.** The flow used by the OAuth2 security scheme. Valid values are `"implicit"`, `"password"`, `"application"` or `"accessCode"`.
-<a name="securitySchemeAuthorizationUrl"></a>authorizationUrl | `string` | `oauth2` (`"implicit"`, `"accessCode"`) | **Required.** The authorization URL to be used for this flow. This SHOULD be in the form of a URL.
-<a name="securitySchemeTokenUrl"></a>tokenUrl | `string` | `oauth2` (`"password"`, `"application"`, `"accessCode"`) | **Required.** The token URL to be used for this flow. This SHOULD be in the form of a URL.
+<a name="securitySchemeAuthorizationUrl"></a>authorizationUrl | `string` | `oauth2` (`"implicit"`, `"accessCode"`) | **Required.** The authorization URL to be used for this flow. This SHOULD be in the form of a URL.  Url may be relative to location of OpenAPI document.
```
Comment (DavidBiesack) [inline versions/3.0.md]: Reads a little awkward an uses Url as a work, not an acronym. I suggest either
"The `authorizationUrl`..." or "The `URL`..."

File: versions/3.0.md [hunk 3]
```diff
@@ -3004,12 +3004,12 @@ Field Name | Type | Validity | Description
...
+<a name="securitySchemeTokenUrl"></a>tokenUrl | `string` | `oauth2` (`"password"`, `"application"`, `"accessCode"`) | **Required.** The token URL to be used for this flow. This SHOULD be in the form of a URL. Url may be relative to location of OpenAPI document.
```
Comment (DavidBiesack) [inline versions/3.0.md]: As above, I suggest either "The `authorizationUrl`..." or "The `URL`..."

**Comment by fehguy** [comment]
@darrelmiller LGTM (XL)

File: versions/3.0.md [hunk 4]
```diff
@@ -3004,13 +3008,10 @@ Field Name | Type | Validity | Description
...
-<a name="securitySchemeScopes"></a>scopes | [Scopes Object](#scopesObject) | `oauth2` | **Required.** The available scopes for the OAuth2 security scheme.
+<a name="securitySchemeScheme"></a>scheme | `string` | `http` | **Required.** The name of the HTTP Authorization scheme to be used in the Authorization header as per [RFC 7235](https://tools.ietf.org/html/rfc7235).
+<a name="securitySchemeBearerFormat"></a>bearerFormat | `string` | `http` (`"bearer"`) | A hint to the client to identify how the bearer token is formatted.  Bearer tokens are usually generated by an authorization server, so this information is primarily for documentation purposes.
+<a name="securitySchemeFlow"></a>flow | [OAuth Flows Object](#oauthFlowsObject) | `oauth2` | **Required.** An object containing configuration information for the flow types supported. 
+<a name="securitySchemeOpenIdConnectUrl"></a>openIdConnectUrl | `string` | `openIdConnect` | **Required.** OpenId Connect URL to discover OAuth2 configuration values. Url may be relative to location of OpenAPI document.
```
Comment (earth2marsh) [inline versions/3.0.md]: Minor typo: `Url` needs all-caps.

**Comment by webron** [comment]
TDC voted for merge, @darrelmiller will fix conflicts and merge when ready.

**Comment by webron** [comment]
@darrelmiller - see if you can fix the conflicts and check @earth2marsh's comment before the call, and we could even merge it live!

File: versions/3.0.md (hunk 1 above)
Comment (darrelmiller) [inline versions/3.0.md]: Done

File: versions/3.0.md (hunk 2 above)
Comment (darrelmiller) [inline versions/3.0.md]: Removed the bit about relative URLs.  Going to try and handle it globally.

File: versions/3.0.md (hunk 3 above)
Comment (darrelmiller) [inline versions/3.0.md]: Removed.

File: versions/3.0.md (hunk 4 above)
Comment (darrelmiller) [inline versions/3.0.md]: Fixed via DELETION :-)

**Comment by webron** [comment]
URLs will change from SHOULD to MUST and then merged :tada:

**Pull request merged**